portfolio/src/main/webapp/script.js

```diff
@@ -34,4 +34,22 @@ function pickRandomQuote() {
 function divMessage() {
     const quoteToDisplay = pickRandomQuote();
     document.getElementById('quote-displayer').innerText = quoteToDisplay;
+}
+
+function fetchMyName() {
+    console.log('Fetching your name...');
+    const responsePromise = fetch('/data');
+    responsePromise.then(handleResponse);
+}
+
+function handleResponse(response) {
+    console.log('Handling response...');
+    const textFromPromise = response.text();
```

Comment:
```suggestion
    const namePromise = response.text();
```

Consider a more specific variable name.